### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by strausr**
[preview](https://github.com/cloudinary/cloudinary_angular/blob/ad1ba188722b59f99dac1571d85abf430a23569b/README.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,63 +1,41 @@
...
-* [Plunker example](https://plnkr.co/edit/Tvrwxt2AW8K5DEhuTKhB?p=preview)
-* [Glitch example](https://glitch.com/edit/#!/cloudinary-angular-sdk)
-
-## Installation ######################################################################
-Install the SDK version that supports the Angular version you are using:
-
-* For Angular 2, use `2.x`
-* For Angular 4, use `4.x`
+## Version Support
 * For Angular versions greater than 5, use `5.x`
```
Comment (maoznir) [inline README.md:29]: Can you please clarify this with more words or add a table? I wouldn't understand what this means.

File: README.md [hunk 2]
```diff
@@ -1,63 +1,41 @@
...
-This SDK is based on the Cloudinary JS module, however the two are decoupled, i.e. this module's `Cloudinary` is a configurable service
-to which you provide your choice of our JS module.
-
-Example Coudinary configuration in your application's module definition:
 
+## Usage
+### Setup
 ```javascript
 import { NgModule } from '@angular/core';
 // ...
```
Comment (maoznir) [inline README.md:41]: Which imports are not shown here?

**Review by maoznir — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (strausr) [reply README.md:29]: updated

File: README.md (hunk 2 above)
Comment (strausr) [reply README.md:41]: related to angular..

File: README.md (hunk 1 above)
Comment (patrick-tolosa) [reply README.md]: Please use a table in this as the format states in our template, we're trying to provide the same look and feel for all SDKs.

File: README.md [hunk 3]
```diff
@@ -1,63 +1,45 @@
...
+- [Transform](https://cloudinary.com/documentation/angular1_video_manipulation#video_transformation_examples) and [optimize](https://cloudinary.com/documentation/angular1_image_manipulation#image_optimizations) assets.
+- Generate [image](https://cloudinary.com/documentation/angular1_image_manipulation#deliver_and_transform_images) and [video](https://cloudinary.com/documentation/angular1_video_manipulation#video_element) tags.
 
-For Angular, Cloudinary provides an SDK for simplifying the integration even further. 
-The SDK serves as a layer on top of one of [Cloudinary's Javascript library](https://github.com/cloudinary/cloudinary_js).
 
+## Version Support
+| SDK Version   | ng 2 | ng 4 | ng >=5.0.0 |
+|---------------|------|------|------------|
+| 2.x           | V    | X    | X          |
```
Comment (patrick-tolosa) [inline README.md:31]: Can you please reverse the order? 
Newest version on top

**Review by patrick-tolosa — APPROVED** [review]

**Review by maoznir — APPROVED** [review]

**Pull request merged**